_posts/publications/2021-01-04-multiphase-swrc.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Investigating the effect of porosity on the soil water retention curve using the multiphase Lattice Boltzmann Method"
+authors: "Hosseini, R., Kumar, K., Delenne, J.Y."
+journal: Powders and Grains 2021, Buenos Aires, Argentina
+layout: publication
+categories: 
+  - publications
+tags:
+  - conference
+  - multiphase-lbm
+  - hpc
+preprint: https://arxiv.org/abs/2102.08420
+abstract:
+The soil water retention curve (SWRC) is the most commonly used relationship in the study of unsaturated soil. In this paper, the effect of porosity on the SWRC is investigated by numerically modeling unsaturated soil using the Shan-Chen multiphase Lattice Boltzmann Method. The shape of simulated SWRCs are compared against that predicted by the van Genuchten model, demonstrating a good fit except at low degrees of saturation. The simulated SWRCs show an increase in the air-entry value as porosity decreases.
```

Comment:
---

## Abstract

The soil water retention curve (SWRC) is the most commonly used relationship in the study of unsaturated soil. In this paper, the effect of porosity on the SWRC is investigated by numerically modeling unsaturated soil using the Shan-Chen multiphase Lattice Boltzmann Method. The shape of simulated SWRCs are compared against that predicted by the van Genuchten model, demonstrating a good fit except at low degrees of saturation. The simulated SWRCs show an increase in the air-entry value as porosity decreases.

Reply:
below the `---`